lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -2570,11 +2601,11 @@ constructTransaction
             . Map.toList
             . foldr (uncurry (Map.insertWith (<>))) Map.empty
 
-decodeValidityInterval
+validValidityInterval
```

Comment:
Again, using `valid` here makes the name a bit hard to understand, IMO.

Perhaps `parse`?

```suggestion
parseValidityInterval
```

Reply:
Ok,

(another idea is to rename it to simple `validityInterval`)